source/install/install-kubernetes-aks.rst

```diff
@@ -0,0 +1,113 @@
+Install Mattermost on Azure AKS
+================================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+You can use are supported `Azure Marketplace Container Offer <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/mattermost.mattermost-operator>`__ to install Mattermost on your existing Azure Infrastructure.  
+
+
+Infrastructure Pre-requisites
+-----------------------------
+
+PostgreSQL v11.0+ Database
```

Comment:
Recommend this to be v13

Reply:
I've applied this feedback.

Reply:
Should we bump it separately along with other instances from these docs that still reflect 11+?

Reply:
I've bumped other references to PG v11+ across the docs site to v12 as part of the v10.1 updates coming this week.